messages/command.md

```diff
@@ -0,0 +1,31 @@
+# RequiresUsernameError
```

Comment:
@WillieRuemmele not sure it will apply to this set of changes, but message files should confirm with message file style guide.